dev/implementations/storage_impl_definitions.h

```diff
@@ -21,11 +21,10 @@ namespace sqlite_orm {
             bool result = false;
             std::stringstream ss;
             ss << "SELECT COUNT(*) FROM sqlite_master WHERE type = " << quote_string_literal("table"s)
-               << " AND name = " << quote_string_literal(tableName);
-            auto query = ss.str();
+               << " AND name = " << quote_string_literal(tableName) << std::flush;
             auto rc = sqlite3_exec(
                 db,
-                query.c_str(),
+                ss.str().c_str(),

```

Comment:
tip: stopping keeping `query` as a dedicated variable makes it harder to debug

Reply:
I thought that the code is simple enough. Note that the stream gets flushed, so you will see the full resulting string in the stringstream variable.